salt/modules/mac_service.py

```diff
@@ -262,17 +313,24 @@ def start(name, runas=None):
     # Get service information and file path
     service = _get_service(name)
     path = service['file_path']
+    label = service['plist']['Label']
+
+    # Enable the service
+    launchctl('enable', 'system/{0}'.format(label), runas=runas)
```

Comment:
Why do we have this line in here? We should only enable a service if explicitly requested, not as a part of starting it